### ISSUE TITLE
RTPEngine_ load testing_log print Issue.

### ISSUE THREAD

**Issue by sitakantaccclogic** [sipwise/rtpengine#1427]
I have tested rtpengine loadtesting in our office lab environment  checked on rtpengine mr 9.5.2.3 versions.

 Rtpengine is installed on AWS  platform  and the instance type is (T2.medium) .
During the testing I had initiated call from one Asterisk to another Asterisk in between Rtpengine was placed and the flow was udp tp tls.all calls are initiate and hangup in a same time.
 In this rtpengine mr 9.5.2.3 I have noticed that after call hangup this can  print the logs in log file up to 70 calls. In the log file noticed that creating new calls 70 and final packet stats 70.
when  I increase the call size( > 70) the final packet stats (i.e media stats which rtp engine used to print for every call)  won't print in log file.

![rtp media stats](https://user-images.githubusercontent.com/97440944/148757164-3cb2a4f7-2dd7-46ea-946f-f93e7ac6d68a.PNG)

Kindly assist.

Thanks in advance
Rgards,
Sitakanta Mohapatra

**Comment by rfuchs** [comment]
I'm not sure I understand the issue. You're saying that when there's more than 70 calls then the final packet stats aren't printed in the log?

**Comment by sitakantaccclogic** [comment]
Yes,the understanding is correct,When the call size is > 70 then all calls packet stats are not pint in the log file.

**Comment by rfuchs** [comment]
I don't really have an answer for you without steps to reproduce. My best guess would be that your syslogd might be rate limiting the log messages? Or perhaps your setup fails to delete the calls fully when there's a higher load. We have installations with much higher call load and haven't seen any problem with the log printing from rtpengine.
